modules/webgl/src/adapter/converters/texture-formats.ts

```diff
@@ -169,7 +169,8 @@ export const TEXTURE_FORMATS: Record<TextureFormat, Format> = {
   'rgb8snorm-webgl': {gl: GL.RGB8_SNORM, b: 3, c: 3, wgpu: false},
 
   // 32-bit formats  
-  'rgba8unorm': {gl: GL.RGBA8, gl1: GL.RGBA, b: 4, c: 2, bpp: 4},
+  'rgba8unorm': {gl: GL.RGBA8, gl1: GL.RGBA, b: 4, c: 2, bpp: 4,
+    dataFormat: GL.RGBA, types: [GL.UNSIGNED_BYTE]},
```

Comment:
Just remember that this table takes up quite a bit of space in the app. While we need what we need, we do want to keep it as slim as possible, e.g. short keys etc.

Reply:
Thanks, this got me on the right track - the `getWebGLTextureParameters` turned out to be the best option, since returns the parameters we need for WebGL here. Was able to remove the changes to the table with that approach!